<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionWriter</class>
 <widget class="QDialog" name="ConditionWriter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save FEM Conditions...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="geoLabel">
     <property name="text">
      <string>Geometry</string>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="condTypeLabel">
     <property name="text">
      <string>Condition Type</string>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="geoBox">
     <item>
      <property name="text">
       <string>All (XML files only)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QComboBox" name="condTypeBox">
     <item>
      <property name="text">
       <string>All (XML files only)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Boundary Conditions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Initial Conditions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Source Terms</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="fileNameLabel">
     <property name="text">
      <string>Filename</string>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="fileNameEdit"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="fileNameButton">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConditionWriter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConditionWriter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
